Nanopi R6: set up reset button to be handled by openwrt

Set up openwrt to handle the reset button appropriately (so that it
can trigger the various recovery modes) on the nanopi R6S and R6C models.

--- a/arch/arm64/boot/dts/rockchip/rk3588s-nanopi-r6.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-nanopi-r6.dtsi
@@ -48,8 +48,8 @@
 		button-user {
 			debounce-interval = <50>;
 			gpios = <&gpio1 RK_PC0 GPIO_ACTIVE_LOW>;
-			label = "User Button";
-			linux,code = <BTN_1>;
+			label = "reset";
+			linux,code = <KEY_RESTART>;
 		};
 	};
 
